From 0bdcf0bb3e0cc0440853bff4aec3eb883923a079 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:27:42 +0200
Subject: [PATCH 1/2] Contacts: Remove unused boolean resource

Fixes:
warn: removing resource com.android.contacts:bool/config_editor_include_phonetic_name
without required default value.

Change-Id: I924a4815d07854ad5ae04ca7ddf87f738a4bd277
---
 res/values-ja/donottranslate_config.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/res/values-ja/donottranslate_config.xml b/res/values-ja/donottranslate_config.xml
index ff8a8eb61..b35785649 100644
--- a/res/values-ja/donottranslate_config.xml
+++ b/res/values-ja/donottranslate_config.xml
@@ -32,7 +32,4 @@
 
     <!-- If true, the order of name fields in the editor is primary (i.e. given name first) -->
     <bool name="config_editor_field_order_primary">false</bool>
-
-    <!-- If true, phonetic name is included in the contact editor by default -->
-    <bool name="config_editor_include_phonetic_name">true</bool>
 </resources>
-- 
2.17.1

